lib/filters/string.js

```diff
@@ -51,6 +51,30 @@ export function isString (value) {
   return _.isString(value)
 }
 
+/**
+ * Get the plural form for an item for a given number of items.
+ *
+ * @see {@link http://blog.gnclmorais.com/simple-pluralise-in-javascript}
+ *
+ * @example
+ * pluralise(1, 'mouse') // 1 mouse
+ * pluralise(2, 'house') // 2 houses
+ * pluralise(2, 'house', { number: false }) // houses
+ * pluralise(2, 'mouse', { plural: 'mice' }) // 10 mice
+ * pluralise(2, 'mouse', { plural: 'mice', number: false }) // mice
+ *
+ * @param {number} count - Number of items
+ * @param {string} singular - Singular form
+ * @param {string} [plural={}] - Plural form
+ * @param {boolean} [number={}] - Output number alongside word
+ * @return {string} Plural form for `count`
+ */
+export function pluralise (count, singular, { plural, number } = {})  {
+  const message = count === 1 ? singular : (plural || `${singular}s`);
+
+  return number === false ? message : `${count} ${message}`;
+}
+
```

Comment:
It's worth pointing out that this is only valid for languages like English with two plural states (`one` and `other`), some languages have multiple words dependent on the number of things (eg `one`, `few`, `many`, `other` – see Russian and Polish as examples)

Reply:
Good point; have added a note to the docs. Might be able to use `Intl` here in the future, but given most services use English, can defer anything more complex